CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 *Enhancements*
 - Uses `TASK_ID` environment variable as the `deployment_id` when rendering resource templates for better [Shipit](https://github.com/Shopify/shipit) integration. ([#430](https://github.com/Shopify/kubernetes-deploy/pull/430))
 
+*Enhancements*
```

Comment:
Since we released 0.25.0 this will need to be moved up

Reply:
Done.

Reply:
Looks like the TASK_ID change didn't actually make it in to the release, but it's in the changelog under that header. Probably needs moving up also.

Reply:
Can you move that as well? Once that's done I'll merge this

Reply:
Done. Good to go.